templates/repo/issue/view_content/pull.tmpl

```diff
@@ -384,7 +390,12 @@
 				{{else if .IsBlockedByRejection}}
 					<div class="item text red">
 						{{svg "octicon-x"}}
-					{{$.i18n.Tr "repo.pulls.blocked_by_rejection"}}
+						{{$.i18n.Tr "repo.pulls.blocked_by_rejection"}}
+					</div>
+				{{else if .IsBlockedByOfficialReviewRequests}}
+					<div class="item text red">
```

Comment:
a small ui bug: ``red`` is invalid.

Reply:
Let's not color these messages. Red icon is enough imho. I only recently removed those colors.